CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - fixed carrying over unexpected guns in holsters to the next level under rare scenarios (#1437, regression from 2.4)
 - fixed item cheats not updating Lara holster and backpack meshes (#1437)
 - improved initial level load time by lazy-loading audio samples (LostArtefacts/TR2X#114)
+- set Lara's default weapon to any weapon she picks up when unarmed, not just pistols (#1443)
```

Comment:
- `added`/`changed`/`fixed`/`removed`/`improved`

I'd say in this change we should go with `changed`